schema/Contact_Candidate.json

```diff
@@ -1,454 +1,394 @@
+                "SAMPLE_DATA": "XYZ Company",
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": "Req if Donor aggregate >$200",
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "occupation": {
+            "title": "OCCUPATION",
+            "description": "",
+            "type": ["string", "null"],
+            "minLength": 0,
+            "maxLength": 38,
+            "pattern": "^[ A-Za-z0-9]{0,38}$",
+            "examples": ["QC Inspector"],
+            "fec_spec": {
+                "COL_SEQ": 14,
+                "FIELD_DESCRIPTION": "OCCUPATION",
+                "TYPE": "A/N-38",
+                "REQUIRED": null,
+                "AUTO_POPULATE": null,
+                "SAMPLE_DATA": "QC Inspector",
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": "Req if Donor aggregate >$200",
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "candidate_office": {
+            "title": "CANDIDATE OFFICE",
+            "description": "",
+            "type": "string",
+            "enum": ["H", "S", "P"],
+            "examples": ["H\nS\nP"],
+            "fec_spec": {
+                "COL_SEQ": 15,
+                "FIELD_DESCRIPTION": "CANDIDATE OFFICE",
+                "TYPE": "Dropdown",
+                "REQUIRED": "X (error)",
+                "AUTO_POPULATE": null,
+                "SAMPLE_DATA": "House\nSenate\nPresidential",
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": null,
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "candidate_state": {
+            "title": "CANDIDATE STATE",
+            "description": "",
+            "type": "string",
+            "minLength": 0,
+            "maxLength": 2,
+            "pattern": "^[ A-Za-z0-9]{0,2}$",
+            "examples": ["WA"],
+            "fec_spec": {
+                "COL_SEQ": 16,
+                "FIELD_DESCRIPTION": "CANDIDATE STATE",
+                "TYPE": "A/N-2",
+                "REQUIRED": "X (error)",
+                "AUTO_POPULATE": null,
+                "SAMPLE_DATA": "WA",
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": "Required if Office = H or S",
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "candidate_district": {
+            "title": "CANDIDATE DISTRICT",
+            "description": "",
+            "type": "string",
+            "minLength": 0,
+            "maxLength": 2,
+            "pattern": "^[ A-Za-z0-9]{0,2}$",
+            "fec_spec": {
+                "COL_SEQ": 17,
+                "FIELD_DESCRIPTION": "CANDIDATE DISTRICT",
+                "TYPE": "A/N-2",
+                "REQUIRED": "X (error)",
+                "AUTO_POPULATE": null,
+                "SAMPLE_DATA": null,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": "Required if Office = H",
```

Comment:
do these rules need to be followed this sprint?

Reply:
No, this rule reference is being covered by ticket web-app#119

Reply:
great!